ios/Podfile

```diff
@@ -42,6 +42,7 @@ target 'KMMIC' do
   end
 
   target 'KMMICKIFUITests' do
+    inherit! :search_paths
```

Comment:
This solved crashes in your case; is that true?

Reply:
@phongvhd93 yessss ❤️